test/functional/tests/data_integrity/test_data_integrity_long.py

```diff
@@ -82,31 +83,44 @@ def test_data_integrity_5d_dss(filesystems):
             sync()
 
     with TestRun.step("Run test workloads on filesystems with verification"):
-        fio_run = Fio().create_command()
+        fio = Fio()
+        fio_run = fio.create_command()
+        fio.base_cmd_parameters.set_param('alloc-size', 1024 * 1024)
         fio_run.io_engine(IoEngine.libaio)
         fio_run.direct()
         fio_run.time_based()
-        fio_run.nr_files(4096)
-        fio_run.file_size_range([(file_min_size, file_max_size)])
         fio_run.do_verify()
         fio_run.verify(VerifyMethod.md5)
         fio_run.verify_dump()
-        fio_run.run_time(runtime)
+        fio_run.run_time(runtime_io)
         fio_run.read_write(ReadWrite.randrw)
         fio_run.io_depth(128)
         fio_run.blocksize_range([(start_size, stop_size)])
         for core in cores:
-            fio_job = fio_run.add_job()
-            fio_job.directory(core.mount_point)
-            fio_job.size(core.size)
-        fio_run.run()
+            start = 512
+            file_section_size = 14 * 1024 * 1024 * 1024  # 14 GiB per IO class
```

Comment:
you can use Size(x, Unit.GibiByte).get_value(Unit.Byte) everywhere instead of "1024 * 1024 * 1024"
If you don't want to use ".get_value(Unit.Byte)" in each calculation, you can make all values in this loop objects of the Size class.